ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -239,6 +251,18 @@
     reason="Test will not run on Openshift dedicated cluster",
 )
 
+skipif_msprovider = pytest.mark.skipif(
```

Comment:
to keep consistency with https://github.com/red-hat-storage/ocs-ci/pull/5473/files#diff-82668ea1726338960f1271091bdb155e25bf053bf5f744a46784269810e5f0b8R215, please call the var `skipif_ms_provider`